src/page/blog/article-page.jsx

```diff
@@ -159,10 +165,47 @@ export const Article = () => {
     });
   };
 
+  const onDelete = () => {
+    if (currentUser.id === data.post.author.id) {
+      // delete blog post from server
+      deletePost({
+        variables: {
+          where: {
+            id: id,
+          },
+        },
+      }).then(() => {
+        // delete blog post from cache
+        client.cache.modify({
+          fields: {
+            posts(existingPosts, { readField }) {
+              return existingPosts.filter(
+                (post) => id !== readField("id", post)
+              );
+            },
+          },
+        });
+        if (type === "edit-published") {
+          router("/articles");
+        } else if (type === "edit-draft") {
+          router(`/draft`);
+        }
+      });
+    }
+  };
+
   return (
     <>
       {!successful ? (
         <div className="relative flex flex-row justify-center mx-auto -top-5 pt-20">
+          {(type === "edit-draft" || type === "edit-published") && (
+            <button
+              onClick={onDelete}
+              className=" absolute top-10 right-10 underline underline-offset-2 text-red-500"
```

Comment:
It would be best to use the tailwind specific styles for this button so that it follows the styles of the other "delete" buttons throughout the app. There are a couple other delete buttons to checkout that should help with this.

Reply:
@fdlai One example is located on the profile-settings-route for that specific delete button. Also, if you checkout the tailwind.config.js file it'll show the terms that are meant to be used throughout the application. For red, it looks like that file shows "danger". I hope that helps!

Reply:
I noticed that there are a couple of different styles for delete buttons in the figma. I was using the styles for the underlined one originally. In this commit, I switched it to the other style, and am now using the "danger" color. I'm not sure if that totally addressed your concern?
![2024-05-01_23-10](https://github.com/jahorwitz/slopopedia/assets/112486038/9103cf0c-3104-4c46-93a3-a40a6bcc7383)
![2024-05-01_23-11](https://github.com/jahorwitz/slopopedia/assets/112486038/2d590bd9-8eb7-4fa5-b824-0f509cbacd46)